test/core/capstanignore_test.go

```diff
@@ -140,12 +140,24 @@ func (s *testingCapstanignoreSuite) TestIsIgnored(c *C) {
 			"additional test #4",
 			"/myfolder/*", "/myfolder2", false,
 		},
+		{
+			"always ignore /meta",
```

Comment:
I was missing another test, for example `/myfolder` and `/myfolder/myfile` but I realised that for this you would need to somehow mimic the fact that `/myfolder` is a directory which is currently not something relevant to `.capstanignore`. I don't think it's mandatory to have it here.

The main reason for thinking about this is because below you are checking that `/meta` is being ignored, but not, for example`, `/meta/package.yaml`.